src/meta-srv/src/metasrv/builder.rs

```diff
@@ -355,6 +358,15 @@ fn build_procedure_manager(
     Arc::new(LocalManager::new(manager_config, state_store))
 }
 
+async fn build_wal_options_allocator(
+    config: &WalConfig,
+    kv_backend: &KvBackendRef,
+) -> Result<WalOptionsAllocator> {
+    let _ = config;
+    let _ = kv_backend;
```

Comment:
SO, it will be implemented in the following PRs?

Reply:
Yes. This PR establishes the skeleton of the remote wal. The following PRs will fill in content.